content/smoketest.md

```diff
@@ -341,5 +341,73 @@ Indicates a section needs work, followed by a description of the task and a prom
 
 To add shortcodes to includes.
 
+### Inline Icons
+
+You can add inline O3DE GUI icons with the `icon` shortcode. Icon `.svg` files are located in `/static/images/icons`.
+
+| Name | Image |
+| - | - |
+| add.svg | {{< icon "add.svg" >}} |
+| animation-editor.svg | {{< icon "animation-editor.svg" >}} |
+| asset-editor.svg | {{< icon "asset-editor.svg" >}} |
+| audio-editor.svg | {{< icon "audio-editor.svg" >}} |
+| camera.svg | {{< icon "camera.svg" >}} |
+| caret-closed.svg | {{< icon "caret-closed.svg" >}} |
+| caret-open.svg | {{< icon "caret-open.svg" >}} |
+| debug.svg | {{< icon "debug.svg" >}} |
+| default-document.svg | {{< icon "default-document.svg" >}} |
+| delete.svg | {{< icon "delete.svg" >}} |
+| entity.svg | {{< icon "entity.svg" >}} |
+| entity-editoronly.svg | {{< icon "entity-editoronly.svg" >}} |
+| entity-notactive.svg | {{< icon "entity-notactive.svg" >}} |
+| entity-outliner-dot.svg | {{< icon "entity-outliner-dot.svg" >}} |
+| file-folder.svg | {{< icon "file-folder.svg" >}} |
+| filter.svg | {{< icon "filter.svg" >}} |
+| grid.svg | {{< icon "grid.svg" >}} |
+| help.svg | {{< icon "help.svg" >}} |
+| helpers.svg | {{< icon "helpers.svg" >}} |
+| landscape-canvas-editor.svg | {{< icon "landscape-canvas-editor.svg" >}} |
+| layer.svg | {{< icon "layer.svg" >}} |
+| level.svg | {{< icon "level.svg" >}} |
+| list-view.svg | {{< icon "list-view.svg" >}} |
+| local.svg | {{< icon "local.svg" >}} |
+| locked.svg | {{< icon "locked.svg" >}} |
+| lua-editor.svg | {{< icon "lua-editor.svg" >}} |
+| material-editor.svg | {{< icon "material-editor.svg" >}} |
+| menu.svg | {{< icon "menu.svg" >}} |
+| more.svg | {{< icon "more.svg" >}} |
+| move.svg | {{< icon "move.svg" >}} |
+| parent.svg | {{< icon "parent.svg" >}} |
+| picker.svg | {{< icon "picker.svg" >}} |
+| pin-button.svg | {{< icon "pin-button.svg" >}} |
+| play.svg | {{< icon "play.svg" >}} |
+| prefab.svg | {{< icon "prefab.svg" >}} |
+| prefab-edit.svg | {{< icon "prefab-edit.svg" >}} |
+| resolution.svg | {{< icon "resolution.svg" >}} |
+| rotate.svg | {{< icon "rotate.svg" >}} |
+| sc-bottom-align.svg | {{< icon "sc-bottom-align.svg" >}} |
+| sc-comment.svg | {{< icon "sc-comment.svg" >}} |
+| sc-group.svg | {{< icon "sc-group.svg" >}} |
+| sc-left-align.svg | {{< icon "sc-left-align.svg" >}} |
+| sc-right-align.svg | {{< icon "sc-right-align.svg" >}} |
+| sc-top-align.svg | {{< icon "sc-top-align.svg" >}} |
+| sc-ungroup.svg | {{< icon "sc-ungroup.svg" >}} |
```

Comment:
Using 'sc' in place of 'script-canvas' might not be intuitive for everyone, especially ESL contributors. I suggest you use 'script-canvas' as the prefix for these files.

Reply:
I have removed the sc prefix and renamed the files. Turns out these icons are general graph canvas icons and are used in other editors, not just Script Canvas.